scripts/alukoayodele.js

```diff
@@ -1,20 +1,22 @@
 
 class Person {
-  constructor( fullName, id, language) {
+  constructor( fullName, id, email, language) {
     
     this.fullName = fullName;
     this.id = id;
+    this.email = email;
     this.language = language;
   }
-  introduction() {
+  output() {
     return `Hello World,This is ${this.fullName} with HNGi7 ID ${this.id} using ${this.language} for stage 2 task.`;
```

Comment:
No email in out put

Reply:
was waiting for the update in the  readme.txt. email added now